deploy/jiva-csi-ubuntu-16.04.yaml

```diff
@@ -0,0 +1,326 @@
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
Lets put a description on the top of the yaml stating what all OSes will it work on

Reply:
sounds good